src/sentry/static/sentry/app/views/groupEventDetails.jsx

```diff
@@ -98,6 +98,13 @@ const GroupEventDetails = React.createClass({
                   occurrences.`)}</span>
               </div>
             }
+            {group.status === 'resolved' && group.statusDetails.inThisRelease &&
+              <div className="alert alert-info alert-block">
+                <span>{t(`This issue has been marked as being resolved in the most
```

Comment:
Anyone have better suggestion for words here? Or do we need to not even show anything special in this case?